src/api/resources/obs_quadrat_benthic_percent.py

```diff
@@ -1,13 +1,28 @@
-from .base import BaseAPIFilterSet, BaseAPISerializer, BaseProjectApiViewSet
 
+from rest_framework.validators import UniqueTogetherValidator
+
+from .base import BaseAPIFilterSet, BaseAPISerializer, BaseProjectApiViewSet
 from ..models import ObsQuadratBenthicPercent
 
 
 class ObsQuadratBenthicPercentSerializer(BaseAPISerializer):
     class Meta:
         model = ObsQuadratBenthicPercent
         exclude = []
-        extra_kwargs = {}
+        extra_kwargs = {
+            "quadrat_number": {
+                "error_messages": {
+                    "null": "Quadrat number is required"
+                }
+            }
+        }
+        validators = [
+            UniqueTogetherValidator(
```

Comment:
Might be good to talk this through... not quite getting why we need a special serializer validator in addition to the submission validations (and/or model `unique_together`), or how they interact...

Reply:
Custom error message wasn't working.  Apparently there was a bug related to this in an earlier version but the version we are currently running should have this fix in it.